src/assets/TestingWithTestCafe/test_cafe_blog.md

```diff
@@ -0,0 +1,162 @@
+---
+title: E2E Testing using TestCafe
+authorName: Sagar Gurung
+authorAvatar: https://avatars.githubusercontent.com/u/46086950?v=4
+authorLink: https://github.com/SagarGi
+createdAt: Nov 20, 2022
+tags: test-cafe, testing, webUI-testing, javascript, automation
+banner: https://raw.githubusercontent.com/DevExpress/testcafe-gh-page-assets/master/src/images/testcafe-ogp-icon.png
+---
+## Background
+
+Are you one of those who is still struggling to write your first End-to-End (E2E) test for your web application?
+
+So what may be the reasons that you are struggling?
+
+- difficult to set up test framework with your web application?
+- difficult to learn?
+- lengthy implementation of code?
```

Comment:
```suggestion
- Lengthy code implementation
```

Reply:
I think here, Sagar is asking questions to the reader